tests/e2e/tests/pilot/access_test.go

```diff
@@ -79,24 +79,24 @@ func (a *accessLogs) check(infra *tutil.Infra) error {
 
 				pod := infra.Apps[app][0]
 				container := inject.ProxyContainerName
-				ns := infra.Namespace
+				ns := infra.Config.Namespace
```

Comment:
can you comment/explain why there is an extra object layer now/what does it buy/help with ?